content/blog/2018/beyondtellerand.md

```diff
@@ -89,6 +89,7 @@ favorites:
 
 {{ embed.from_event(
   collections.all,
+  page=talk,
```

Comment:
When more than one event has the same "venue" it breaks our `embed.from_event` macro unless `page` is specified. This fixes it for this case, but maybe we want something more robust?